<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachCoreDialog</class>
 <widget class="QDialog" name="AttachCoreDialog">
  <property name="windowTitle">
   <string>Start Debugger</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="execLabel">
       <property name="text">
        <string>&amp;Executable:</string>
       </property>
       <property name="buddy">
        <cstring>execFileName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Utils::PathChooser" name="execFileName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="coreLabel">
       <property name="text">
        <string>&amp;Core file:</string>
       </property>
       <property name="buddy">
        <cstring>coreFileName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Utils::PathChooser" name="coreFileName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="toolchainLabel">
       <property name="text">
        <string>&amp;Tool chain:</string>
       </property>
       <property name="buddy">
        <cstring>toolchainComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Debugger::Internal::DebuggerToolChainComboBox" name="toolchainComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebuggerToolChainComboBox</class>
   <extends>QComboBox</extends>
   <header>debuggertoolchaincombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
